examples/example_utils.cpp

```diff
@@ -17,73 +17,74 @@ mip::Timestamp getCurrentTimestamp()
 }
 
 
-std::unique_ptr<ExampleUtils> openFromArgs(const std::string& port_or_hostname, const std::string& baud_or_port, const std::string& binary_file_path)
+std::unique_ptr<ExampleUtils> openFromArgs(const std::string& port_or_hostname, const std::string& baud_or_port,
+    const std::string& binary_file_path)
 {
     auto example_utils = std::unique_ptr<ExampleUtils>(new ExampleUtils());
 
-    if( !binary_file_path.empty() )
+    if (!binary_file_path.empty())
     {
 #ifdef MIP_USE_EXTRAS
         example_utils->recordedFile = std::unique_ptr<std::ofstream>(new std::ofstream(binary_file_path));
-        if( !example_utils->recordedFile->is_open() )
+        if (!example_utils->recordedFile->is_open())
             throw std::runtime_error("Unable to open binary file");
-#else  // MIP_USE_EXTRAS
+#else // !MIP_USE_EXTRAS
         throw std::runtime_error("The program was compiled without binary file recording support. Recompile with -DMIP_USE_EXTRAS=ON");
```

Comment:
I prefer to just use the original condition (or a shorthand version) in the comment. This keeps it consistent with the comment on the #endif. It can also help prevent it from going stale when the original condition is updated (say an additional condition was added).